services/CollectionService/config/settings.yml

```diff
@@ -1,8 +1,11 @@
 islandora:
+  apiVersion: 0.0.1
   fedoraProtocol: http
```

Comment:
@ruebot, just an idea i had. Since our api is tied to branch name right now maybe that numbering makes no sense. Any suggestions? thanks!